Tasks/Common/VstsAzureHelpers_/Utility.ps1

```diff
@@ -5,14 +5,14 @@
         [Switch] $ServicePrincipal
     )
 
-    # Add the certificate to the cert store.
-    $certificate = New-Object System.Security.Cryptography.X509Certificates.X509Certificate2
```

Comment:
Is it currently broken in V3 task?If not why this change is required for V4 task?

Reply:
Even with my change, it is working fine when using powershell.exe
Problem is coming when using pwsh.exe, somehow it is not able to access the import method, so I made this change. 
You can refer this : https://www.cloudnotes.io/x509certificate-is-immutable-on-this-platform-use-the-equivalent-constructor-instead/